doc/dev/repo-access.rst

```diff
@@ -7,21 +7,22 @@ Special branches
 * ``master``: current tip (integration branch)
 * ``next``: pending release (feature frozen, bugfixes only)
 * ``last``: last/previous release
-* ``dumpling``, ``cuttlefish``, ``bobtail``, ``argonaut``, etc.: stable release branches
-* ``dumpling-next``: backports for stable release, pending testing
+* Release branches (for example ``luminous``)
```

Comment:
Release branches (`luminous`, `kraken`, etc.) corresponding to the releases listed at `doc/releases.rst`